src/types/Cohort.ts

```diff
@@ -12,3 +12,11 @@ export const CohortPhases: CohortPhaseType[] = [
   'Phase 2 - Plan and Scale',
   'Phase 3 - Implement and Monitor',
 ];
+
+/**
+ * Get string value of phase number.
+ */
+export const getPhaseNumber = (phase: CohortPhaseType): string => {
```

Comment:
The table renders phases as numbers instead of the full phase name. I have a question on figma suggesting we show the full phase (like we do on the cohorts list).